lib/iris/fileformats/__init__.py

```diff
@@ -28,12 +28,9 @@
 from . import abf
 from . import um
 try:
-    import iris_grib as igrib
+    from . import grib as igrib
```

Comment:
gribapi isn't available for python 3 or windows (which is one of the reasons we want to move to eccodes).

This try/except mimics what we had in version 1.9.2 before we took out iris_grib (see [here](https://github.com/SciTools/iris/blob/v1.9.2/lib/iris/fileformats/__init__.py#L30))